### PR TITLE
rent lead form base setup

### PR DESCRIPTION
**Pull request by tadjohnston**
[Story](https://rentpath.leankit.com/card/583330299)

- setup rent lead form
- added base styling
- setup story using react-themed

screen shot 2017-12-04 at 4 21 26 pm

### CODE REVIEW

**Comment by tadjohnston** [comment]
@mneumark yeah, but I didn't know if I wanted to add them yet because I don't know how much things will change here.  I can add a few though if we'd like.

**Comment by ninomichaels** [comment]
looks great 👍

File: packages/react-ui-rent/src/LeadForm/__tests__/LeadForm-test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+describe('LeadForm', () => {
+  describe('fields', () => {
+    let wrapper
+
+    beforeEach(() => {
+      wrapper = mount(<LeadForm />)
+    })
+
+    it('has necessary form field names', () => {
+      expect(wrapper.find('input[name="name"]').length).toBe(1)
```
Comment (mneumark) [inline packages/react-ui-rent/src/LeadForm/__tests__/LeadForm-test.js]: Can you change this to:
```js
expect(wrapper.find('input[name="name"]')).toHaveLength(1)
```

**Comment by lfender6445** [comment]
Excellent work 👍

**Comment by jtheodore-rent** [comment]
👍

**Pull request merged**